src/net_processing.cpp

```diff
@@ -5694,17 +5694,19 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
                     LOCK(tx_relay->m_bloom_filter_mutex);
 
                     for (const auto& txinfo : vtxinfo) {
-                        const uint256& hash = peer->m_wtxid_relay ? txinfo.tx->GetWitnessHash() : txinfo.tx->GetHash();
+                        const uint256& hash = peer->m_wtxid_relay ? txinfo.tx->GetWitnessHash().ToUint256() : txinfo.tx->GetHash().ToUint256();
                         CInv inv(peer->m_wtxid_relay ? MSG_WTX : MSG_TX, hash);
                         tx_relay->m_tx_inventory_to_send.erase(hash);
+
+                        tx_relay->m_tx_inventory_to_send.erase(inv.hash);
```

Comment:
Isn't this the same as `tx_relay->m_tx_inventory_to_send.erase(hash);` two lines up?

Reply:
whoops, removed